### PR TITLE
[WIP] Add conv2d benchmark for ONNX Runtime

### PR DESCRIPTION
**Pull request by axmat**

### CODE REVIEW

File: examples/conv-opt/comparison/gen_conv_models.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                      [6, 5, 4, 3, 0, -3, -4, -5, -6],
+                      [7, 6, 5, 4, 0, -4, -5, -6, -7],
+                      [8, 7, 6, 5, 0, -5, -6, -7, -8],
+                      [7, 6, 5, 4, 0, -4, -5, -6, -7],
+                      [6, 5, 4, 3, 0, -3, -4, -5, -6],
+                      [5, 4, 3, 2, 0, -2, -3, -4, -5],
+                      [4, 3, 2, 1, 0, -1, -2, -3, -4]], dtype='float32')
+}
+
+def get_output_shape(i):
```
Comment (zhanghb97) [inline examples/conv-opt/comparison/gen_conv_models.py:32]: Use two spaces indentation for consistency, just like the onnxruntime-conv2d.py file.

File: examples/conv-opt/comparison/gen_conv_models.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
+import onnx
```
Comment (zhanghb97) [inline examples/conv-opt/comparison/gen_conv_models.py:1]: Add an "onnx" prefix and use the hyphen "-" in the file name for consistency.

File: examples/conv-opt/comparison/README.md [hunk 3]
```diff
@@ -33,6 +33,13 @@ $ cd buddy-mlir/examples/conv-opt/comparison/
 $ python3 pytorch-conv2d.py
 ```
 
+- ONNX Runtime
+
+```
+$ cd buddy-mlir/examples/conv-opt/comparison/
+$ python3 onnxruntime-conv2d.py
```
Comment (zhanghb97) [inline examples/conv-opt/comparison/README.md:40]: It seems the onnx model generation step needs to be added here.

**Review by zhanghb97 — CHANGES_REQUESTED** [review]
Thanks! I have left some comments. And for the onnx model generation, please remove the binary files and add the generation step in the README file (let users generate those models themselves). Don't forget to add the *.onnx to the `.gitignore`.

File: examples/conv-opt/comparison/gen_conv_models.py (hunk 2 above)
Comment (axmat) [reply examples/conv-opt/comparison/gen_conv_models.py]: Added onnx prefix for helper and make_tensor_value_info. Unfortunately, onnx doesn't export the numpy_helper attribute.

File: examples/conv-opt/comparison/README.md (hunk 3 above)
Comment (axmat) [reply examples/conv-opt/comparison/README.md:51]: Added step to generate onnx models.

**Comment by zhanghb97** [comment]
Thanks!

**Pull request merged**